packages/cli/package.json

```diff
@@ -217,6 +217,11 @@
             "type": "object",
             "description": "Listening configuration for local development",
             "properties": {
+              "https": {
+                "type": "boolean",
+                "visibility": "frontend",
+                "description": "The protocol that the frontend should be use. Only used for local development."
```

Comment:
Perhaps something like "Whether the frontend should be served over HTTPS. ..." could work a bit better here? Fits a bit better with it being a bool